modules/firefox/each-config.nix

```diff
@@ -0,0 +1,154 @@
+# manually applied args
+{
+  mkTarget,
+  name,
+  humanName,
+}:
+# module args
+{
+  lib,
+  pkgs,
+  ...
+}:
+mkTarget {
+  inherit name humanName;
+
+  extraOptions = {
+    profileNames = lib.mkOption {
+      description = "The ${humanName} profile names to apply styling on.";
+      type = lib.types.listOf lib.types.str;
+      default = [ ];
+    };
+
+    colorTheme.enable = lib.mkEnableOption "[Firefox Color](https://color.firefox.com/) on ${humanName}";
+
+    firefoxGnomeTheme.enable = lib.mkEnableOption "[Firefox GNOME theme](https://github.com/rafaelmardojai/firefox-gnome-theme) on ${humanName}";
+  };
+
+  configElements = [
+    (
+      { cfg }:
+      {
+        warnings =
+          lib.optional (cfg.profileNames == [ ])
```

Comment:
This used to _also_ be predicated on `config.programs.${target.path}.enable` (i.e. `programCfg.enable`). Was/is that somehow redundant or incorrect?